libs/SalesforceSDK/src/com/salesforce/androidsdk/security/SalesforceKeyGenerator.java

```diff
@@ -190,4 +221,17 @@ private static String getAddendum(String name) {
         final String suffix = TextUtils.isEmpty(name) ? "" : name;
         return String.format(Locale.US, ADDENDUM, suffix);
     }
+
+    @Nullable
+    private static KeyPair getRSA2048KeyPair() {
+        KeyPair kp = null;
```

Comment:
I think this can be moved into the `else` block in the other method. Not a whole lot of value in splitting this piece out as a `private` method.